bowling-ball/Game.cs

```diff
@@ -1,19 +1,62 @@
-﻿using System;
+using System;
 
 namespace BowlingBall
 {
     public class Game
     {
-        public void Roll(int pins)
-        {
-            throw new NotImplementedException();
-        }
+        Frames [] frames = new Frames[10];
+        int PointerToFrame;
+        const int TotalPossibleFrames = 10;
+        int ActualFrames;

```

Comment:
what is the use of `ActualFrames` and `PointerToFrame`? 
avoid creating unnecessary data members for a class.